go/src/koding/kites/kloud/provider/google/imgfamily_test.go

```diff
@@ -0,0 +1,158 @@
+package google
+
+import (
+	"reflect"
+	"testing"
+)
+
+var getFromFamilyFixture = map[string]string{
+	"ubuntu-os-cloud/ubuntu-1404-lts":            "ubuntu-1404-trusty-v20161010",
+	"ubuntu-os-cloud/ubuntu-1604-lts":            "ubuntu-1604-xenial-v20161013",
+	"windows-sql-cloud/sql-std-2016-win-2012-r2": "sql-2016-standard-windows-2012-r2-dc-v20161012",
+	"rhel-cloud/rhel-7":                          "rhel-7-v20160921",
+	"coreos-cloud/coreos-stable":                 "coreos-stable-1122-2-0-v20160906",
+}
+
+var getFromFamilyTestFunc = func(project, family string) string {
+	return getFromFamilyFixture[project+"/"+family]
+}
+
+func TestFamily2Image(t *testing.T) {
+	tests := []struct {
```

Comment:
nitpick: use `map[string]struct{ ...` instead of `[]struct{ Name string; ...`, you get test-case randomisation for free